docs/source/en/quicktour.mdx

```diff
@@ -425,6 +387,53 @@ Create a model from your custom configuration with [`TFAutoModel.from_config`]:
 
 Take a look at the [Create a custom architecture](./create_a_model) guide for more information about building custom configurations.
 
+## PyTorch Trainer
+
+All models are a standard [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) or a [`tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) so you can use them in any standard training loop. While you can write your own training loop, 🤗 Transformers provides a [`Trainer`] class for PyTorch, which contains the basic training loop and adds additional functionality for features like distributed training, mixed precision, and more.
+
+<Tip>
+
+For TensorFlow, use [`~TFPreTrainedModel.prepare_tf_dataset`] to prepare your dataset as a `tf.data.Dataset`. Then you can [`compile`](https://keras.io/api/models/model_training_apis/#compile-method) and [`fit`](https://keras.io/api/models/model_training_apis/#fit-method) your model with the usual Keras methods. Take a look at the [finetuning tutorial](./training#convert-dataset-to-tensorflow-format) for more details.
+
+</Tip>
+
+Depending on your task, you'll typically pass the following parameters to [`Trainer`]:
+
+1. A [`PreTrainedModel`] or a [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module).
+2. [`TrainingArguments`] contains the model hyperparameters you can change like learning rate, batch size, and the number of epochs to train for. The default values are used if you don't specify any training arguments.
+3. Your train and test datasets.
+4. A preprocessing class like a tokenizer, feature extractor, or processor.
+5. A [`DataCollator`] to create a batch of examples from your dataset.
+
+```py
+>>> from transformers import Trainer
+
+>>> trainer = Trainer(
+...     model=model,
+...     args=training_args,
+...     train_dataset=dataset["train"],
+...     eval_dataset=dataset["test"],
+...     tokenizer=tokenizer,
+...     data_collator=data_collator,
+... )
+```
+
+When you're ready, call [`~Trainer.train`] to start training:
+
+```py
+>>> trainer.train()
+```
+
+<Tip>
+
+For tasks that use a sequence-to-sequence model like translation or summarization, use the [`Seq2SeqTrainer`] and [`Seq2SeqTrainingArguments`] classes instead.
+
+</Tip>
+
+You can customize the training loop behavior by subclassing the methods inside [`Trainer`]. This allows you to customize features such as the loss function, optimizer, and scheduler. Take a look at the [`Trainer`] reference for which methods can be subclassed. 
+
+The other way to customize the training loop is by using [Callbacks](./main_classes/callbacks). You can use callbacks to integrate with other libraries and inspect the training loop to report on progress or stop the training early. Callbacks do not modify anything in the training loop itself. To customize something like the loss function, you need to subclass the [`Trainer`] instead.
```

Comment:
That's more for the `Trainer` page, but I'd showcase an example of subclassing each method. I think there's a single one shown for `compute_loss`, but subclassing requires understanding which inputs/outputs will work and it's not necessarily straightforward for a beginner